cypress/integration/integration_tests/nav_test.js

```diff
@@ -0,0 +1,15 @@
+describe('Integration test for the navbar', () => {
+  it('Open the nav menu and change pages', () => {
+    cy.visit('');
+
+    // Click the burger.
+    cy.get('#nav-input').click();
+
+    // Click the add asset page link. This is forced because Cypress mistakenly
+    // believes the element is not visible.
```

Comment:
does it change if you do cy.awaitLoad()? Just to understand why.

Reply:
No it doesn't unfortunately. It's because the `<a>` tag's dimensions are 0x0. I guess I could fiddle with the CSS more but I'm not sure if it's worth it.

Reply:
Definitely not worth it :) just wanted to understand the cause! Maybe add a few words to the comment saying this about the <a> tag?